src/content/api/notifications.mdoc

```diff
@@ -0,0 +1,44 @@
+---
+title: Notifications
+description: Notification models
+nav:
+  path: Notifications
+  order: 1
+---
+A Notification is sent to subscribers when specific events occur in our system. Notifications may contain details about the event, and relevant metadata.
+
+## Notification Model
+
+{% properties %}
+  {% property name="id" type="string" %}
+    Notification's unique identifier.
+  {% /property %}
+
+  {% property name="type" type="string" %}
+    The event that occurred that triggered the Notification. See below for the list of types.
+  {% /property %}
+
+  {% property name="data" type="string" %}
+    The associated metadata.
```

Comment:
@Ray2126 Suggestion: Provide more context here - this is a very generic description